query/sql/sql_instance_postgresql_log_statement_database_flag_set_appropriately.sql

```diff
@@ -0,0 +1,22 @@
+select 
+  -- Required Columns
+  self_link resource,
+  case
+    when database_version not like 'POSTGRES%' then 'skip'
+    when database_flags is null or not (database_flags @> '[{"name":"log_statement"}]') then 'alarm'
+    when database_flags @> '[{"name": "log_statement","value":"ddl"}]' then 'ok'
+    else 'alarm'
+  end status,
```

Comment:
```suggestion
  end as status,
```
Please make this change to all places.